assets/dash.css

```diff
@@ -2135,3 +2135,61 @@ float: none;
 justify-content: center;
 }
 }
+
+
+/* Sara's corner of the Dash Stylesheet :) */
+
+/* esports design tweaks */
+.page-id-67894 .btn.btn-white{
+background: #f8f9fa;
+border-radius: 4px;
+}
+
+.page-id-67894 .container-sm{
+width: 100%;
+}
+
+#why-dash-esports .image{
+text-align: center;
+}
+
+.page-id-67894 .container-xs {
+    width: 100%;
+    padding-left: 0;
+    padding-right: 15px;
+}
+
+.page-id-67894 .container-xs .richtext{
+max-width: 480px;
+}
+
+.page-id-67894 .card-item{
+text-align: left;
+padding: 40px;
+position: relative;
+padding-bottom: 30px;
+}
+
+.page-id-67894 .card-item h4{
+min-height: 70px;
+}
+
+.page-id-67894 .card-item img{
+max-width: 100%;
+}
+
+.page-id-67894{
+  .banner .caption{
+  width: auto!important;
+  }
+}
```

Comment:
Nesting CSS selectors like this and in the following section is invalid, this might work in some browsers but behaviour will be inconsistent. Please use a linter to validate your CSS :+1:

Reply:
There are some rules in the code above that use nesting, that's why I tried it! I'll adjust my code to not be nested. :)

Reply:
Check it closely - it is just fancy syntactical use of indentation, the brackets are not actually nested inside one another. The "nested" selector still specifies the superordinate `.stat-item` selector with nested indentation, like in this example:
```css
.stat-item {
  color: #031841; }
  .stat-item .value {
    display: block;
    font-weight: 200;
    color: #008de4;
    font-size: 54px; }
```

Reply:
Aha! A while back I used to create nested CSS, but it was saved as a different type of file. I don't remember what it was called (sass maybe), but I think there was something running in the background that converted it to normal CSS for the browser. So when I saw what looked like nesting, I was excited to use that method because it's a bit faster.

Reply:
Yeah, SASS is a variation of the CSS language that gets compiled down to normal CSS (I think). There are SASS files in the theme, so maybe you can figure out how they work - I never looked at it...